pkg/minikube/cluster/commands.go

```diff
@@ -43,3 +43,5 @@ func GetStartCommand() string {
 	flags := strings.Join(flagVals, " ")
 	return fmt.Sprintf(startCommandFmtStr, flags, constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath)
 }
+
+var localkubeStatusCommand = fmt.Sprintf("ps aux | grep localkube | wc -l")
```

Comment:
Done.  
I was not able to do this as cleanly as I'd liked, for whatever reason running ps through ssh command would not let me use the -p or --no-headers flag as I was going to do originally:
var localkubeStatusCommand = fmt.Sprintf(`
sudo sh -c 'PATH=/usr/local/bin/ps:$PATH ps -p $(%s) --no-headers | wc -l'
`, constants.LocalkubePIDPath)

Now I am using:
var localkubeStatusCommand = fmt.Sprintf(`
ps aux | grep $(cat %s)| wc -l
`, constants.LocalkubePIDPath)